pkg/cmd/server/api/validation/master.go

```diff
@@ -192,5 +192,12 @@ func ValidatePolicyConfig(config api.PolicyConfig) fielderrors.ValidationErrorLi
 func ValidateProjectRequestConfig(config api.ProjectRequestConfig) fielderrors.ValidationErrorList {
 	allErrs := fielderrors.ValidationErrorList{}
 
+	if len(config.ProjectRequestTemplate) > 0 {
```

Comment:
If ProjectRequestTemplate is required, shouldn't you throw an error if it's absent?